src/App.jsx

```diff
@@ -14,70 +16,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function toogleSelect(selectGood) {
+    if (selectGood === good) {
+      setGoods('');
+    } else {
+      setGoods(selectGood);
+    }
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <h1 className="title is-flex is-align-items-center">
+        {good.length > 0 ? `${good} is selected` : 'No goods selected'}
```

Comment:
a state can store an empty string
and to check for an empty string, you can simply cast it to a boolean value, since an empty string in a boolean representation is a `false` like this:
`good ? () : ()`